apps/judicial-system/web/src/routes/Prosecutor/Indictments/Overview/Overview.tsx

```diff
@@ -302,6 +297,16 @@ const Overview: FC = () => {
             </BlueBox>
           </Box>
         )}
+        <Box component="section" marginBottom={10}>
+          <SectionHeading
```

Comment:
Nit: Consider moving the section heading into `InputPenalties` to avoid duplicated sections content